diff --git a/MANIFEST.in b/MANIFEST.in
index dc3a643d..132681a3 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,4 +1,4 @@
 include brax/tests/testdata/cylinder.stl
 include brax/v2/envs/assets/*.xml
-recursive-include brax/v2/test_data *.xml *.stl *.obj
-recursive-include brax/v2/visualizer *
\ No newline at end of file
+recursive-include brax/v2/test_data *.xml *.stl *.obj *.urdf
+recursive-include brax/v2/visualizer *
